gradle.properties

```diff
@@ -29,6 +29,7 @@ kotlin.native.ignoreDisabledTargets=true
 kotlin.mpp.enableCompatibilityMetadataVariant=false
 # https://youtrack.jetbrains.com/issue/KT-45545#focus=Comments-27-4773544.0-0
 kapt.use.worker.api=false
+kotest.proptest.default.iteration.count=10
```

Comment:
Oof, this is a very low number. Any reason why? Any reason why this was introduced here? 🤔

Reply:
In the older tests this was set using a subclass of `StringSpec`, so I changed to the recommended variant. However, I forgot to change the number 🙈

Reply:
Done! Now it's using Kotest's default